app/src/main/java/io/pumpkinz/pumpkinreader/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package io.pumpkinz.pumpkinreader.model;
+
+import java.util.Date;
+
+
+public abstract class Item {
```

Comment:
Couldn't we just use the ItemPOJO for the base class, or Item as Retrofit container?
I think this violates DRY.